Created separate props for QuickSearch and SearchPage inputs

liwzhang opened the pull request:
Fixes #146. Also fixes the error where clearing one input clears both inputs.
Requires `add-search-prop` branch in conveyor-redux and autoinvent-example.

taroken6 reviewed (approved):
Copy-pasta since I commented on issue instead:

All changes on autoinvent-example, Conveyor, and Conveyor-Redux works on my end. Code looks good too.
One minor change I have in mind for Autoinvent-example is remove the unnecessary State props in Search Container. Like how Quick Search has [searchPageQueryText](https://gitlab.moesol.com/myurchyshyna/autoinvent-example/-/blob/add-search-prop/src/containers/SearchContainer.js#L23) and Search Page has [quickSearchQueryText](https://gitlab.moesol.com/myurchyshyna/autoinvent-example/-/blob/add-search-prop/src/containers/SearchContainer.js#L23)

liwzhang commented:
> Copy-pasta since I commented on issue instead:
> 
> All changes on autoinvent-example, Conveyor, and Conveyor-Redux works on my end. Code looks good too.
> One minor change I have in mind for Autoinvent-example is remove the unnecessary State props in Search Container. Like how Quick Search has [searchPageQueryText](https://gitlab.moesol.com/myurchyshyna/autoinvent-example/-/blob/add-search-prop/src/containers/SearchContainer.js#L23) and Search Page has [quickSearchQueryText](https://gitlab.moesol.com/myurchyshyna/autoinvent-example/-/blob/add-search-prop/src/containers/SearchContainer.js#L23)

That's a good catch, I've removed them! While I did that I found an issue in conveyor, which I also just fixed
